src/server.c

```diff
@@ -5669,9 +5669,9 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "server_name:%s\r\n", SERVER_NAME,
                 "valkey_version:%s\r\n", VALKEY_VERSION,
                 "valkey_release_stage:%s\r\n", VALKEY_RELEASE_STAGE,
-                "redis_git_sha1:%s\r\n", serverGitSHA1(),
-                "redis_git_dirty:%i\r\n", strtol(serverGitDirty(), NULL, 10) > 0,
-                "redis_build_id:%s\r\n", serverBuildIdString(),
+                "server_git_sha1:%s\r\n", serverGitSHA1(),
```

Comment:
Some part of me says let's just leave them. One more thing to not list as a breaking change.

Reply:
Yes, it's a breaking change.

So I think we will not change it in 9.0 then. But I'm wondering: How long are we going to keep everything 'redis'? Forever? Five years? If we keep it forever, then users will always remember that Redis was the original and Valkey is just a fork.

If we want to accept the risk, we are allowed to make breaking changes in a major version. I understand "redis_version" is used by many clients, but the git fields are likely not used by many. If we decide to change it some time, I hope users will understand that we want to phase out 'redis' named fields.